exe/ruby-lsp-doctor

```diff
@@ -1,23 +1,5 @@
 #!/usr/bin/env ruby
 # frozen_string_literal: true
 
-$LOAD_PATH.unshift(File.expand_path("../lib", __dir__))
-require "ruby_lsp/internal"
-
-if File.exist?(".index.yml")
-  begin
-    config = YAML.parse_file(".index.yml").to_ruby
-  rescue => e
-    abort("Error parsing config: #{e.message}")
-  end
-  RubyIndexer.configuration.apply_config(config)
-end
-
-index = RubyIndexer::Index.new
-
-puts "Globbing for indexable files"
-
-RubyIndexer.configuration.indexables.each do |indexable|
-  puts "indexing: #{indexable.full_path}"
-  index.index_single(indexable)
-end
+puts "The ruby-lsp-doctor command is deprecated. Please run `ruby-lsp --doctor` instead."
```

Comment:
I vote for getting rid of this file. It's just used for troubleshooting anyway, it's not going to be super disruptive for users.

Let's just ensure it's not referenced in any docs or instructions.

Reply:
Any strong opinion @st0012 ?

Reply:
Yeah let's remove it 👍 Remember to update `gemspec` too.